docs/getting_started.coffee.md

```diff
@@ -183,6 +183,67 @@ capitalized.
   * `CreationDate` - the date the document was created (added automatically by PDFKit)
   * `ModDate` - the date the document was last modified
 
+## Encryption and Access Privileges
+
+PDF specification allow you to encrypt the PDF file and require a password when opening the file,
+and/or set permissions of what users can do with the PDF file. PDFKit implements standard security
+handler in PDF version 1.3 (40-bit RC4), version 1.4 (128-bit RC4), PDF version 1.7 (128-bit AES),
+and PDF version 1.7 ExtensionLevel 3 (256-bit AES).
+
+To enable encryption, provide a user password when creating the `PDFDocument` in `options` object.
+The PDF file will be encrypted when a user password is provided, and users will be prompted to enter
+the password to decrypt the file when opening it.
+
+ * `userPassword` - the user password (string value)
+
+To set access privileges for the PDF file, you need to provide an owner password and permission
+settings in the `option` object when creating `PDFDocument`. By default, all operations are disallowed.
+You need to explicitly allow certain operations.
+
+ * `ownerPassword` - the owner password (string value)
+ * `allowPrinting` - whether printing is allowed. Specify `"lowResolution"` to allow degraded printing, or `"highResolution"` to allow printing with high resolution
+ * `allowModifying` - whether modifying the file is allowed. Specify `true` to allow modifying document content
+ * `allowCopying` - whether copying text or graphics is allowed. Specify `true` to allow copying
+ * `allowAnnotating` - whether annotating, form filling is allowed. Specify `true` to allow annotating and form filling
+ * `allowFillingForms` - whether form filling and signing is allowed. Specify `true` to allow filling in form fields and signing
+ * `allowContentAccessibility` - whether copying text for accessibility is allowed. Specify `true` to allow copying for accessibility
+ * `allowDocumentAssembly` - whether assembling document is allowed. Specify `true` to allow document assembly
```

Comment:
Should not be better a permissions hash? I.E.:

```
{
 permissions: {
   printing: '',
   copying: true
   etc
 }
}
```

Reply:
For the record, these are some the naming used by other API:

http://itextsupport.com/apidocs/itext7/7.0.1/com/itextpdf/kernel/pdf/EncryptionConstants.html
https://pdfbox.apache.org/docs/1.8.13/javadocs/

Seems there's not a uniform nomenclature

Reply:
Hash is a better idea.